projects/shared/src/lib/class/user.ts

```diff
@@ -111,6 +111,16 @@ export class User {
     this.currentOrganisation = organisation;
   }
 
+  /**
+   * User symbol name
+   */
+  get symbolName() {
+    return [
+      this.first_name[0],
+      this.last_name[0],
+    ].join('').toUpperCase();
```

Comment:
According to PO, the firstname could be none. Maybe a test should be necessary here ?

Reply:
done